packages/affiliate/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "automattic/jetpack-affiliate",
+	"description": "Reads an affiliate code for Jetpack. Useful to add it to the connection URL so it's read by wpcom.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"version": "1.0.0",
+	"require": {},
+	"require-dev": {
+		"phpunit/phpunit": "^5.7 || ^6.5 || ^7.5",
+		"php-mock/php-mock": "^2.1"
```

Comment:
It doesn't seem like we're using `phpmock` in this package?

Reply:
Good catch.